packages/yourloops/components/patient/add-dialog.tsx

```diff
@@ -121,23 +119,24 @@ function AddDialog(props: AddDialogProps): JSX.Element {
     }
   }
 
-  const dialogIsOpen = props.actions !== null
-  const buttonAddDisabled = errorMessage !== null || !isValidEmail() || teamId.length < 1
-  const optionsTeamsElements: JSX.Element[] = [
-    <MenuItem id="patient-list-dialog-add-team-option-none" aria-label={t('none')} value="" key="none" />
-  ]
-  const teams = props.actions?.teams ?? []
-  for (const team of teams) {
-    optionsTeamsElements.push(
-      <MenuItem
-        id={`patient-list-dialog-add-team-option-${team.name}`}
-        value={team.id}
-        key={team.id}
-        aria-label={team.name}
-      >
-        {team.name}
-      </MenuItem>
-    )
+  const addPatient = async (): Promise<void> => {
+    try {
+      setInProgress(true)
+      const team = teams.find(team => team.id === teamId)
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**🤓 nitpick:** ‏There is a `getTeam` method in the team hook